### PR TITLE
DOC: explain changes to boxplot styles

### PR DESCRIPTION
**Pull request by phobson**
did i do this right?

### CODE REVIEW

File: doc/users/dflt_style_changes.rst [hunk 1]
```diff
@@ -405,6 +405,73 @@ or by setting ::
...
+
+For the new defaults, everything is black except for the median and mean
+lines (if drawn), which are set to the first two elements of the current
+color cycle. Also, the default flier markers are now hollow circles,
+which maintain the ability of the plus-symbols to overlap without
+obscuring data too much.
+
+.. plot::
+
+    fig, (old, new) = pyplot.subplots(ncols=2, sharey=True)
```
Comment (tacaswell) [inline doc/users/dflt_style_changes.rst]: Where is data defined?

Comment (tacaswell) [inline doc/users/dflt_style_changes.rst]: These code blocks are run as part of the doc-build process to generate the example images.

Comment (phobson) [inline doc/users/dflt_style_changes.rst]: added it in, sorry about that

Comment (tacaswell) [inline doc/users/dflt_style_changes.rst]: no worries, thanks for you work!

**Pull request merged**